starlette/concurrency.py

```diff
@@ -13,6 +13,16 @@
 T = typing.TypeVar("T")
 
 
+async def run_until_first_complete(*args: typing.Tuple[typing.Callable, dict]) -> None:
+    async def run(handler: typing.Callable[..., typing.Coroutine]) -> None:
```

Comment:
```suggestion
    async def run(handler: typing.Callable[[], typing.Coroutine]) -> None:
```

Reply:
Yeah just added that lol, thanks.

Reply:
643d107a88f1e26b0c0485b8ca155488f75fabf1